packages/katello/katello/katello.spec

```diff
@@ -18,20 +18,19 @@ Source1:    katello-debug.sh
 Source2:    katello-remove
 Source3:    katello-remove-orphans
 Source4:    katello-service
-Source5:    service-wait
-Source6:    katello-restore
-Source7:    katello-backup
-Source8:    katello-service-bash_completion.sh
-Source9:    qpid-core-dump
-Source10:   katello-clean-empty-puppet-environments
-Source11:   katello-change-hostname
-Source12:   katello-repository-publish-check
-Source13:   katello-change-hostname.8.asciidoc
-Source14:   restore.rb
-Source15:   backup.rb
-Source16:   hostname-change.rb
-Source17:   helper.rb
-Source18:   katello-backup-rotate-tar.sh
+Source5:    katello-restore
+Source6:    katello-backup
+Source7:    katello-service-bash_completion.sh
+Source8:    qpid-core-dump
+Source9:   katello-clean-empty-puppet-environments
+Source10:   katello-change-hostname
+Source11:   katello-repository-publish-check
+Source12:   katello-change-hostname.8.asciidoc
+Source13:   restore.rb
+Source14:   backup.rb
+Source15:   hostname-change.rb
+Source16:   helper.rb
+Source17:   katello-backup-rotate-tar.sh
```

Comment:
Note this shift of source numbers means you have to update all references to these in the spec file. For example, checkout line 109 or 108.

Reply:
I'd change `Source17` to `Source5` and keep the diff small. Perhaps you could even leave out `Source5` and not change anything about these files. A smaller change means its easier to review, easier cherry picks and much smaller risk of regressions because of it.